mava/configs/system/q_learning/rec_iql.yaml

```
@@ -31,3 +31,7 @@ gamma: 0.99  # discount factor
 
 eps_min: 0.05
 eps_decay: 1e5
+
+# --- Sebulba parameters ---
+replay_ratio : 4  # Average number of times the learner should sample each item from the replay buffer.
+error_tolerance:  ~ # Tolerance for how much the learner/actor can sample/insert before being blocked. Must be greater than 2 to avoid deadlocks.
```

I think this is a slightly better explainer

```suggestion
error_tolerance: float  # Maximum allowed deviation from the target sampling ratio. Controls how far the actual samples can vary from the ideal samples-per-insert ratio. Must be > 2 to prevent deadlocks.
```

Also let's give this a value, because I think the blocking one might be too blocking so would be nice to make this the default. A value of ~4 seems good to me what did you find in your experiments?

---

I think 4 is reasonable, Most of the top SMAC runs had 2 but some had 16